### PR TITLE
Add new filter bytes diagram, improve table, see #11

### PR DESCRIPTION
**Pull request by svgeesus**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2937,30 +2944,41 @@ <h2>Filter types for filter method
...
 <tr>
-<td class="Regular"><tt>c</tt> </td>
+<td class="Regular">c </td>
 <td class="Regular">the byte corresponding to b in the pixel immediately before the pixel containing b (or the byte immediately before b, when the bit depth is less than 8).</td>
 </tr>
 </table>
 
-<p><a href="#paethpredictor-function"></a> shows the relative positions of the bytes <tt>x</tt>,
+<figure id="integer-representation-in-png">
+  <object id="figure72" data="figures/filter-bytes.svg" type="image/svg+xml" style="width: 12vw">
```
Comment (ProgramMax) [inline index.html:2976]: Since this is a new figure, can we go with an id that isn't related to the section? Perhaps a descriptive name.

**Review by ProgramMax — COMMENTED** [review]
LGTM but can we change the figure id?

File: index.html (hunk 1 above)
Comment (svgeesus) [reply index.html:2976]: Sure! It wasn't clear to me how the figure id and the object id were different.

Comment (svgeesus) [reply index.html:2976]: Oh I see, sorry that was a dumb copy-paste error to get the markup, from another figure!

Comment (ProgramMax) [reply index.html:2976]: Oh sorry. I do like the figure id more now. But I actually meant the object id. Sorry about that.

What I meant is "figure72" sounds like it assumes what section it is in.

I don't think the object needs an id at all. The figure does so we can link to it with ReSpec magic. I suspect elsewhere the figure and object both have ids so we can preserve an old fragment identifier while also giving it a new-style name.

I think in this case we can just drop the object id.

**Comment by svgeesus** [comment]
@ProgramMax said:

> LGTM but can we change the figure id?

Done with https://github.com/w3c/PNG-spec/pull/148/commits/15ff9efda2e259da873ff554557a29c084f7707d ok now?

**Review by ProgramMax — APPROVED** [review]
No no, that commit updated the figure's id, not the object's id.
(That said, this isn't critical and can be fixed later or ignored if we don't care enough. I'll go ahead and approve.)

**Pull request merged**